setup.py

```diff
@@ -3,13 +3,14 @@
 """Setup configuration."""
 
 from setuptools import setup, find_packages
+import ephypype
 
+VERSION = ephypype.__version__
```

Comment:
you may need to copy this: https://github.com/mne-tools/mne-python/blob/master/setup.py#L13-L20